content/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/.conf.json

```diff
@@ -0,0 +1,20 @@
+{
+  "order": [
+    "Introduction",
+    "Paths_for_Supporting_U2F_or_FIDO2_Security_Keys",
+    "Security_Key_Compatibility",
+    "Known_Issues",
+    "FAQ"
+
```

Comment:
there's an extra blank line here that can be dropped

Reply:
Removing extra blank line that was line 8